app/views/play/CampaignView.js

```diff
@@ -921,12 +925,31 @@ module.exports = (CampaignView = (function () {
         const campaignSlug = window.location.pathname.split('/')[2]
         return this.promptForSubscription(campaignSlug, 'premium campaign visited')
       }
+
+      if (
+        !me.get('email') &&
+        (storage.load('prompted-for-signup') || storage.load('prompted-for-subscription'))
```

Comment:
we would want to show ai-league and roblox even if they haven't signed up yet so that visibility of products increases

Reply:
@mrfinch it's not really clear what you mean here.

Reply:
We show roblox modal if they have signed up by the email check you added. We should promote ai league and roblox even if they haven't signed up.